src/test/java/org/tailormap/api/controller/GeoServiceProxyControllerIntegrationTest.java

```diff
@@ -341,6 +350,60 @@ void test_obk_wmts_GetTile_Conditional() throws Exception {
         .andExpect(status().isNotModified());
   }
 
+  @Test
```

Comment:
please also add one or more testcase that check you are getting an "unauthorized" response when no user is provided, see eg. https://github.com/Tailormap/tailormap-api/blob/fa19837db154eaf0fb61c2509ae8a3ad54a453c2/src/test/java/org/tailormap/api/controller/GeoServiceProxyControllerIntegrationTest.java#L102